app/src/main/java/de/westnordost/streetcomplete/quests/localized_name/AddBusStopName.java

```diff
@@ -0,0 +1,88 @@
+package de.westnordost.streetcomplete.quests.localized_name;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddBusStopName extends SimpleOverpassQuestType
+{
+
+	@Inject public AddBusStopName(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override protected String getTagFilters()
+	{
+		return "nodes with" +
+			" ((public_transport=platform and (bus=yes or trolleybus=yes or tram=yes))" +
```

Comment:
It should be renamed from "AddBusStopName.java" or trams should not be accepted here (the first solution is probably preferable).

Note that also, question, commit description and icon may require changes.

Reply:
Icon does not, in the other quests the bus also stands for any public transport. (see the crossed part in https://github.com/westnordost/StreetComplete/pull/986#issuecomment-375926979, where I already mentioned that.)

Also still see https://github.com/westnordost/StreetComplete/pull/986#discussion_r176920870 BTW, you do not exclude the stop position.

Reply:
Does anyone care?

Reply:
Ah, it's actually below this line, sorry for the noise. Just did not see this was already resolved.